cmd/functions.go

```diff
@@ -0,0 +1,256 @@
+package main
+
+var (
+	// functions it's the list of fuctions that will be added
+	// to the AWSReader with the corresponding implementation
+	functions = []Function{
+		// ec2
+		Function{
+			Entity:  "Instances",
+			Prefix:  "Describe",
+			Service: "ec2",
+			Documentation: `
+			// GetInstances returns all EC2 instances based on the input given.
+			// Returned values are commented in the interface doc comment block.
+			`,
+		},
+		Function{
+			Entity:  "Vpcs",
+			Prefix:  "Describe",
+			Service: "ec2",
+			Documentation: `
+			// GetVpcs returns all EC2 VPCs based on the input given.
+			// Returned values are commented in the interface doc comment block.
+			`,
+		},
+		Function{
+			Entity:  "Images",
+			Prefix:  "Describe",
+			Service: "ec2",
+			Documentation: `
+			// GetImages returns all EC2 AMI belonging to the Account ID based on the input given.
+			// Returned values are commented in the interface doc comment block.
+			`,
+		},
+		Function{
+			Entity:  "SecurityGroups",
+			Prefix:  "Describe",
+			Service: "ec2",
+			Documentation: `
+			// GetSecurityGroups returns all EC2 security groups based on the input given.
+			// Returned values are commented in the interface doc comment block.
+			`,
+		},
+		Function{
+			Entity:  "Subnets",
+			Prefix:  "Describe",
+			Service: "ec2",
+			Documentation: `
+			// GetSubnets returns all EC2 subnets based on the input given.
+			// Returned values are commented in the interface doc comment block.
+			`,
+		},
+		Function{
+			Entity:  "Volumes",
+			Prefix:  "Describe",
+			Service: "ec2",
+			Documentation: `
+			// GetVolumes returns all EC2 volumes based on the input given.
+			// Returned values are commented in the interface doc comment block.
+			`,
+		},
+		Function{
+			Entity:  "Snapshots",
+			Prefix:  "Describe",
+			Service: "ec2",
+			Documentation: `
+			// GetSnapshots returns all snapshots belonging to the Account ID based on the input given.
+			// Returned values are commented in the interface doc comment block.
+			`,
+		},
+
+		// elasticache
```

Comment:
are those commentary still relevant?

Reply:
It's to keep the list somehow grouped, with AWS Services.

Reply:
OK